.editorconfig

```diff
@@ -474,6 +474,16 @@ dotnet_diagnostic.CA1859.severity = none
 # https://learn.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5104
 dotnet_diagnostic.NU5104.severity = none
 
+# SYSLIB1045: Use 'GeneratedRegexAttribute' to generate the regular expression implemenatation at compile-time
+# Justification: Would not have much impact on us / is only available in .NET8
+# https://learn.microsoft.com/dotnet/fundamentals/syslib-diagnostics/syslib1045
+dotnet_diagnostic.SYSLIB1045.severity = none
```

Comment:
A few of the analyzers being suppressed here are because they involve using functions exclusive to .NET8. My thought process is that splitting up the logic would be harder to maintain. Let me know if this is the wrong approach. The other errors being suppressed here come from #406 which I had reverted but just brought back.